src/applications/vaos/components/calendar/CalendarCell.jsx

```diff
@@ -111,7 +110,6 @@ const CalendarCell = ({
           currentlySelectedDate={date}
           additionalOptions={additionalOptions}
           handleSelectOption={handleSelectOption}
-          optionsError={optionsError}
```

Comment:
When I initially created the Calendar Widget, I thought we would be displaying validation errors within the calendar row.  New design calls for it at the top of the widget, so I removed all references to `optionsError` and old code to display error messages.